rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -2986,10 +2985,11 @@ public String getLoginFromSessionId(@PathParam("sessionId") String sessionId) {
     public UserData getUserDataFromSessionId(@PathParam("sessionId") String sessionId) {
         if (sessionId != null && sessionStore.exists(sessionId)) {
             try {
+                renewSession(sessionId);
                 Scheduler scheduler = sessionStore.get(sessionId).getScheduler();
                 UserData userData = scheduler.getCurrentUserData();
                 return userData;
-            } catch (NotConnectedException e) {
+            } catch (Exception e) {
```

Comment:
If we catch all exception here we should at least log something otherwise it will be hard to find the issue if there is a bug.